README.md

```diff
@@ -172,6 +172,19 @@ OPTIONS:
 EXAMPLE: run.sh -s "Squeak-trunk" --headfull /path/to/projects
 ```
 
+## Running locally
+
+If you need to run smalltalkCI locally you can follow these steps:
+
+1. clone your repository
+  * a) from github: `git clone https://github.com/your/project.git`, or
+  * b) clone your local repo: `git clone /wherever/is/your/project`, or
+  * c) use your local repo directly
+2. cd into the cloned directory `cd project`
+3. download smalltalkCI: `wget -O smalltalkCI.zip https://github.com/hpi-swa/smalltalkCI/archive/master.zip`
+4. unzip it: `unzip -o smalltalkCI.zip`
+5. run smalltalkCI, provide dialect you want to test and your local config: `smalltalkCI-master/run.sh --headfull -s "Pharo-5.0" .smalltalk.ston`
+
```

Comment:
No, because you are instructing folks to drop a big fat directory into their checkout and you should finish the job by having a cleanup step ... 

If it were up to me I would not drop the directory there --- there is really nothing to be gained by dropping smalltalkCI into a persons checkout ... smalltalkCI works fine from anywhere --- the project under test directory is calculated relative to the .smalltalk.ston file

If you have folks install smalltalkCI in a neutral location it can be used for multiple projects and/or over a long period of time and for multiple travis error conditions over time ... dropping smalltalkCI into a checkout means that it will have to be deleted before the next commit of their project something that will happen as soon as they solve their original issue ...